wordpress-mobile/gutenberg-mobile
# Analytics - Add support to register block insertions from the clipboard

**Related PRs:**
- https://github.com/WordPress/gutenberg/pull/60626
- https://github.com/wordpress-mobile/WordPress-Android/pull/20617
- https://github.com/wordpress-mobile/WordPress-iOS/pull/22982

## To test:

> [!NOTE]  
> Use the following test builds:
> * [iOS](https://github.com/wordpress-mobile/WordPress-iOS/pull/22982#issuecomment-2047022821)
> * [Android](https://github.com/wordpress-mobile/WordPress-Android/pull/20617#issuecomment-2047042126)

- Open the app
- Create a new post
- Paste some content

<details>
<summary><b>Example content</b></summary>

**NOTE:** Make sure to copy the text from the "Copy" button.
```html
<h1>Ea dolores aliquid ut voluptatum ipsa. </h1><p>Lorem ipsum dolor sit amet. Est maiores sequi et optio commodi <strong>Non consectetur sed sapiente quia</strong> et sequi officiis et dolores praesentium et facilis repellendus. At saepe voluptatem <a href="https://www.loremipzum.com" target="_blank">[Sed adipisci ab iste odio](https://www.loremipzum.com/)</a>. Sit facilis minus rem incidunt istein omnis in distinctio laudantium. Et autem architectoAb consequatur nam ullam neque vel corporis dolores. </p><ol><li>Ea enim officiis ex molestias neque id rerum unde. </li><li>Et facere repellendus non pariatur laudantium id sint tenetur. </li></ol><dl><dt><dfn>Ut magni quae. </dfn></dt><dd>Hic dolores galisum aut atque dicta est internos fugiat. </dd><dt><dfn>Ea repellat modi! </dfn></dt><dd>Non natus laudantium non quaerat possimus et repudiandae sunt qui ullam galisum. </dd><dt><dfn>Et nobis voluptatem. </dfn></dt><dd>Ut necessitatibus nihil ut optio nesciunt ea sint rerum est laboriosam reiciendis. </dd></dl><blockquote cite="https://www.loremipzum.com">Non molestiae voluptas 33 soluta illo et reiciendis recusandae et pariatur libero sit doloribus ullam. </blockquote><h2>Et laboriosam libero est incidunt architecto ut Quis repellendus. </h2><p>Id cupiditate exercitationem et rerum doloremqueaut iste 33 minus delectus. Qui autem repudiandae eos consequuntur explicaboin sint ut laudantium expedita sed nesciunt necessitatibus. Non natus illo <em>Non quam ea nemo temporibus non voluptate nulla</em> eos cumque quis in enim optio. </p><h3>Sit nesciunt ipsum aut provident distinctio? </h3><p>Ut voluptatibus itaque <em>Vel dolorem qui asperiores enim non nesciunt error</em> rem saepe voluptas sit voluptas culpa et neque consequatur. Non impedit quam <strong>Ex architecto et illum vitae ex quas animi</strong>. Non debitis quae <a href="https://www.loremipzum.com" target="_blank">[Est ipsa et quibusdam minus qui veritatis aliquid cum dignissimos quia](https://www.loremipzum.com/)</a> in accusamus reprehenderit et odit vitae! Aut explicabo dolorem a porro excepturiAb quis ut temporibus eaque ex optio enim vel enim earum. </p><ul><li>Ex atque laboriosam cum facere eveniet aut omnis enim et quis architecto! </li><li>Eos fuga nostrum est deleniti voluptatem. </li><li>Rem numquam dolorum et cupiditate sapiente sit rerum perspiciatis! </li><li>Et vitae debitis non eveniet ipsum. </li></ul>
```

</details>

- **Expect** to see the `editor_block_inserted` events with the following data:
- `block_name`, `blocks_replaced: true` and `insert_method: clipboard`

PR submission checklist:
